packages/js-config-webpack-plugin/package.json

```diff
@@ -0,0 +1,81 @@
+{
+    "name": "js-config-webpack-plugin",
+    "version": "0.3.0",
+    "description": "",
+    "main": "src/index.js",
+    "license": "MIT",
+    "author": "Namics AG - Technology & Application",
+    "contributors": [
+        {
+            "name": "Jan Biasi",
+            "email": "biasijan@gmail.com",
+            "url": "https://github.com/janbiasi"
+        },
+        {
+            "name": "Matthias Giger",
+            "email": "matthias.giger@namics.com",
+            "url": "https://github.com/naminho"
+        },
+        {
+            "name": "Davide De Giorgio",
+            "email": "dave_degiorgio@hotmail.com",
+            "url": "https://github.com/degiorgioo"
+        },
+        {
+            "name": "Ernst Ammann",
+            "email": "ernscht@gmail.com",
+            "url": "https://github.com/ernscht"
+        },
+        {
+            "name": "Jan Nicklas",
+            "email": "j.nicklas@me.com",
+            "url": "https://github.com/jantimon"
+        }
+    ],
+    "engines": {
+        "node": ">=8 <11",
+        "npm": ">=5"
+    },
+    "scripts": {
+        "test": "jest --runInBand --verbose --forceExit",
+        "posttest": "tsc",
+        "test-watch": "jest --watch --verbose --forceExit",
+        "update-snapshots": "jest -u --runInBand"
+    },
+    "peerDependencies": {
+        "typescript": ">=2.0.0",
```

Comment:
why do we need typescript here?

Reply:
Is not needed. It was a copy-paste error.